[BUG] Build fails in TypeScript 5.7-dev (nightly)

### Software versions

TypeScript 5.7.0-pr-58573-47 (soon to be merged)

### Browser name and version

_No response_

### Jupyter notebook / Jupyter Lab version

_No response_

### Expected behavior

Build completes successfully.

### Observed behavior

Build fails with the following errors due to out-of-date types for `@types/node`:
```
TS2769 at /bokehjs/test/devtools/baselines.ts#L46
		error TS2769: No overload matches this call.
	TS2322 at /bokehjs/test/devtools/baselines.ts#L128
		error TS2322: Type 'Buffer' is not assignable to type 'Uint8Array<ArrayBufferLike>'.
	TS2322 at /bokehjs/test/devtools/baselines.ts#L128
		error TS2322: Type 'Buffer' is not assignable to type 'Uint8Array<ArrayBufferLike>'.
	TS2345 at /bokehjs/test/devtools/devtools.ts#L629
		error TS2345: Argument of type 'Buffer' is not assignable to parameter of type 'string | ArrayBufferView | Stream | Iterable<string | ArrayBufferView> | AsyncIterable<string | ArrayBufferView>'.
	TS2345 at /bokehjs/test/devtools/devtools.ts#L641
		error TS2345: Argument of type 'Buffer' is not assignable to parameter of type 'Uint8Array<ArrayBufferLike>'.
```

### Example code

```Python
1. Clone repository.
2. Update bokeh/bokehjs/make/package.json to use the following package for `"typescript"`: "typescript": "npm:@typescript-deploys/pr-build@5.7.0-pr-58573-47" (see https://github.com/microsoft/TypeScript/pull/58573#issuecomment-2375395832)
3. Perform build.
```

TypeScript 5.7 will introduce a breaking change to the `Uint8Array`, `ArrayBuffer`, and `SharedArrayBuffer` types that requires [update definitions](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/70694) for `@types/node`, version 22.7.4 or later.


### Stack traceback or browser console output

_No response_

### Screenshots

_No response_

--- a/bokehjs/make/package.json
+++ b/bokehjs/make/package.json
@@ -13,7 +13,7 @@
   ],
   "devDependencies": {
     "@types/eslint": "^8.56.7",
-    "@types/node": "^22.5.5",
+    "@types/node": "^22.7.4",
     "@types/which": "^3.0.3",
     "@types/yargs": "^17.0.32",
     "acorn": "^8.12.1",
@@ -24,8 +24,8 @@
     "semver": "^7.6.2",
     "ts-node": "^10.9.2",
     "tsconfig-paths": "^4.2.0",
-    "typescript-eslint": "^7.16.0",
     "typescript": "~5.6.2",
+    "typescript-eslint": "^7.16.0",
     "which": "^3.0.1",
     "yargs": "^17.7.2"
   }

--- a/bokehjs/src/compiler/package.json
+++ b/bokehjs/src/compiler/package.json
@@ -13,7 +13,7 @@
     "@types/convert-source-map": "^2.0.3",
     "@types/css": "^0.0.37",
     "@types/less": "^3.0.6",
-    "@types/node": "^22.5.5",
+    "@types/node": "^22.7.4",
     "@types/yargs": "^17.0.32",
     "chalk": "^4.1.2",
     "combine-source-map": "^0.8.0",

--- a/bokehjs/src/server/package.json
+++ b/bokehjs/src/server/package.json
@@ -9,7 +9,7 @@
     "url": "https://github.com/bokeh/bokeh.git"
   },
   "devDependencies": {
-    "@types/node": "^22.5.5",
+    "@types/node": "^22.7.4",
     "@types/ws": "^8.5.10",
     "@types/yargs": "^17.0.32",
     "chalk": "^4.1.2",

--- a/bokehjs/test/package.json
+++ b/bokehjs/test/package.json
@@ -15,7 +15,7 @@
     "@types/cli-progress": "^3.11.5",
     "@types/cors": "^2.8.17",
     "@types/express": "^4.17.21",
-    "@types/node": "^22.5.5",
+    "@types/node": "^22.7.4",
     "@types/nunjucks": "^3.2.6",
     "@types/pngjs": "^6.0.4",
     "@types/sinon": "^17.0.2",

